examples/charts/bar.py

```diff
@@ -15,20 +20,69 @@
 silver = np.array([data[abbr]['silver'] for abbr in countries], dtype=np.float)
 bronze = np.array([data[abbr]['bronze'] for abbr in countries], dtype=np.float)
 
+# Dict input
+
 medals = OrderedDict(bronze=bronze, silver=silver, gold=gold)
 
-bar1 = Bar(medals, countries, title="grouped, dict_input", xlabel="countries", ylabel="medals",
-          legend=True, width=800, height=600)
+dict_stacked = Bar(
+    medals, countries, title="OrderedDict input | Stacked", legend=legend_position,
+    xlabel="countries", ylabel="medals", width=width, height=height,
+    stacked=True
+)
+
+#dict_grouped = Bar(
+#    medals, countries, title="OrderedDict input | Grouped", legend=legend_position,
+#    xlabel="countries", ylabel="medals", width=width, height=height
+#)
 
-bar2 = Bar(medals, countries, title="stacked, dict_input", xlabel="countries", ylabel="medals",
-          legend=True, width=800, height=600, stacked=True)
+# Data frame input
 
 df = pd.DataFrame(medals, index=countries)
-bar3 = Bar(df, title="stacked, df_input", xlabel="countries", ylabel="medals", legend=True, width=800, height=600, stacked=True)
 
-medals = np.array([bronze, silver, gold])
-bar4 = Bar(df, title="grouped, df_input", xlabel="countries", ylabel="medals", legend=True, width=800, height=600)
+#df_stacked = Bar(
```

Comment:
I left it in there, because I was removing examples from the output that were there before, but its probably a bad idea because they could easily get out of date without anyone noticing. Thanks for the spot, will update along with the validation.